### PR TITLE
Change the entities used in the Syndicate Base

### LINKED ISSUE #8280: Fake windows smooth with riveted walls
window_smoothing

FYI, fake windows are technically walls. We could use real windows too, but this allows us to minimize atmos simulation on Z-level 2

### PR DESCRIPTION
**Pull request by ghost**
- Outside now uses the actual snow tile from Arctic Station instead of the old "fake snow" tile. Since the turfs in the Outpost are unsimulated, no risk of the cold air getting in
- Replace the old and broken "fake windows" with actual windows
- Tweaked the "Blocker" entity, renamed invisible wall and now has alpha = 0 so you can't actually see it, prevents Nuke Ops from leaving the base through the windows
- Added the brand new radial generators, they behave pretty correctly and should be fun

Fixes #8280 since it removes the fake walls

Tested, looks good. The map wasn't changed in any gameplay measure, only the outside scenery. In the future and if I can find the strength to code in more Arctic stuff, might make the scenery a bit better

### CODE REVIEW

**Comment by PJB3005** [comment]
> Tweaked the "Blocker" entity, renamed invisible wall and now has alpha = 0

I see no code changes.
Subtypes you goof. 

Also make sure to give them 101 invisibility.

**Comment by ComicIronic** [comment]
> real snow

What's the performance hit from this?

**Comment by ghost** [comment]
Zero performance hit, real snow is unsimulated and there's no simulated turfs on the Z-level

**Comment by ghost** [comment]
Alright, removed subtypes and all

File: code/modules/spells/aoe_turf/conjure/forcewall.dm [hunk 1]
```diff
@@ -51,3 +51,19 @@
...
+/obj/effect/forcefield/invisible_wall
+	name = "invisible wall"
+	desc = "The nemesis of explorers since the dawn of times."
+
+/obj/effect/forcefield/invisible_wall/New()
+
+	..()
+
+	alpha = 0
+	invisibility = 101
```
Comment (PJB3005) [inline code/modules/spells/aoe_turf/conjure/forcewall.dm:58]: ... And why don't you just set these in the type definition?

File: code/modules/spells/aoe_turf/conjure/forcewall.dm [hunk 2]
```diff
@@ -51,3 +51,19 @@
...
+
+//This is a system object, don't cultify it
+/obj/effect/forcefield/invisible_wall/cultify()
+
```
Comment (PJB3005) [inline code/modules/spells/aoe_turf/conjure/forcewall.dm]: If you're gonna whitespace at least be consistent with it.

File: code/modules/spells/aoe_turf/conjure/forcewall.dm [hunk 3]
```diff
@@ -51,3 +51,19 @@
 	new /obj/effect/forcefield/cult(get_turf(src))
 	qdel(src)
 	return
+
+/obj/effect/forcefield/invisible_wall
+	name = "invisible wall"
+	desc = "The nemesis of explorers since the dawn of times."
+
+/obj/effect/forcefield/invisible_wall/New()
+
```
Comment (PJB3005) [inline code/modules/spells/aoe_turf/conjure/forcewall.dm]: Same.

**Comment by PJB3005** [comment]
You should definitely do this for the other maps too.

**Comment by Konater** [comment]
I couldn't get dream daemon to start when there were too many snow tiles involved, are you sure there won't be any issues?

File: code/modules/spells/aoe_turf/conjure/forcewall.dm [hunk 4]
```diff
@@ -51,3 +51,17 @@
...
 	return
+
+/obj/effect/forcefield/invisible_wall
+	name = "invisible wall"
+	desc = "The nemesis of explorers since the dawn of times."
+
+/obj/effect/forcefield/invisible_wall/New()
+	..()
+
+	alpha = 0
```
Comment (ComicIronic) [inline code/modules/spells/aoe_turf/conjure/forcewall.dm]: Why do you need to set this stuff in New()?

Comment (ghost) [inline code/modules/spells/aoe_turf/conjure/forcewall.dm]: Prevents it from being invisible in the map editor due to alpha = 0

You want to actually see the invisible walls when you're working with them

**Comment by ghost** [comment]
Might have to do with something too expensive in the snow turf's New() or Init() causing the server startup proc to hard crash (above 5 minutes)

File: code/modules/spells/aoe_turf/conjure/forcewall.dm (hunk 4 above)
Comment (PJB3005) [inline code/modules/spells/aoe_turf/conjure/forcewall.dm]: I believe invisibility is irrelevant to the mapper, so removing alpha and setting invisibility to max would do the job.

**Comment by clusterfack** [comment]
Real snow is shit for map placed objects because they have smoothing

**Comment by ghost** [comment]
No idea, but it loads fine when testing

**Comment by ghost** [comment]
Really, it should be fine. Didn't take ten hours to load in, everything is still fine

Maybe there's a small performance hit on loading only, but I can't benchmark it. Would require reworking snow tiles too

**Comment by PJB3005** [comment]
> Really, it should be fine. 
> 
> Should
> 
> Dylan

**Comment by ghost** [comment]
I've tested it, ran it, no significant compile time increase, run time delays, and zero lag. I've also tested making sure you can't get out into the cold snow, or get cold air to seep in by breaking the windows

**Comment by PJB3005** [comment]
This begs the question. 

Why would you change it.

Theres no reason for those tiles to not be fake.

**Comment by ghost** [comment]
They are unsimulated and thus don't cause any atmos, power or lighting calculations

And they work for the purpose of generating a snow environment, along with the snow radial gens that only work on that tile type

**Comment by ghost** [comment]
Sincerely hope it's conflicting because someone redid my changes, because I can't be arsed right now

**Pull request closed without merging**